warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -1868,6 +1876,11 @@ protected ASTJexlScript upperCaseIdentifiers(MetadataHelper metadataHelper, Shar
             Sets.newHashSet(excerptFields.getFields()).stream().forEach(s -> excerptFields.replace(s, s.toUpperCase()));
         }
 
+        SummarySize summarySize = config.getSummarySize();
+        if (summarySize != null && !summarySize.isEmpty()) {
+            summarySize.getContentNames().stream().forEach(s -> summarySize.replace(s, s.toUpperCase()));
+        }
```

Comment:
Uppercasing the content names should happen internally to SummarySize anytime new names are added.

Reply:
fixed